docker-compose.yml

```
@@ -0,0 +1,56 @@
+  userauthapi:
+    image: userauthapi:latest
+    build:
+      context: ./AuthAndUserAPI
+      dockerfile: Dockerfile
+    container_name: userauthapi
+    ports:
+      - "7065:8080"  # Korrigeret portbinding
+    environment:
+      - ASPNETCORE_ENVIRONMENT=Docker
+      - Jwt__Key=${JWT_KEY}
+      - ConnectionStrings__DefaultConnection=Data Source=/app/AuthDb.sqlite; 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)